File: API/WebApplication1/Models/Fravaer.cs
namespace WebApplication1.Models
{
    public class Fravaer : Common
    {
        public required DateTime Start { get; set; }
        public required DateTime End { get; set; }
        public required string MedarbejderId { get; set; }
        public Medarbejder? Medarbejder { get; set; }
    }

    public class FravaerDTO
    {
        public required DateTime Start { get; set; }
        public required DateTime End { get; set; }
        public required string MedarbejderId { get; set; }
        public Medarbejder? Medarbejder { get; set; }
    }
}

File: Flutter/assignly/lib/classes/objects/fravaer.dart
import 'package:assignly/classes/objects/medarbejder.dart';
import 'dart:convert';

class Fravaer {
  String? id, medarbejderId;
  DateTime? start, end;
  Medarbejder? medarbejder;

  static Fravaer getFravaerFromJson(String response) {
    try {
      final data = jsonDecode(response);
      final results = data;

    } catch (_) {
    }
    return Fravaer();
  }
}